lib/Doctrine/ORM/Query.php

```diff
@@ -596,7 +596,7 @@ public function free(): void
     /**
      * Sets a DQL query string.
      *
-     * @param string $dqlQuery DQL Query.
+     * @param string|null $dqlQuery DQL Query.
```

Comment:
This method is a no-op when `null` is passed. Should we rather deprecate passing `null` so that we can use `string` as native parameter type in 3.0?

Reply:
> Should we rather deprecate passing null so that we can use string as native parameter type in 3.0?

Yes, but not rather. We should do so on 2.13.x.